src/Tools/BuildValidator/Extensions.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
 using System.Reflection.Metadata;
 using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
```

Comment:
```suggestion
```

Reply:
I do not think you introduced usages of types in this namespace.

Reply:
Most of these weren't needed. Deleted a bunch locally.